### PR TITLE
update usingTellor for tellorX

### PR DESCRIPTION
**Pull request by tkernell**

### CODE REVIEW

**Comment by themandalore** [comment]
can we get the checks to pass?  Or at least take them off if it doesn't work with the upgrade?

**Comment by tkernell** [comment]
It should be good to go. Took out some more clutter.

File: contracts/interface/ITellor.sol [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    function verify() external pure returns(uint);
+    function vote(uint256 _disputeId, bool _supports, bool _invalidQuery) external;
+    function voteFor(address[] calldata _addys,uint256 _disputeId, bool _supports, bool _invalidQuery) external;
+    function getDelegateInfo(address _holder) external view returns(address,uint);
+    function isFunctionApproved(bytes4 _func) external view returns(bool);
+    function getVoteRounds(bytes32 _hash) external view returns(uint256[] memory);
+    function getVoteCount() external view returns(uint256);
+    function getVoteInfo(uint256 _id) external view returns(bytes32,uint256[9] memory,bool[2] memory,VoteResult,bytes memory,bytes4,address[2] memory);
+    function getDisputeInfo(uint256 _id) external view returns(uint256,uint256,bytes memory, address);
+    function getOpenDisputesOnId(uint256 _queryId) external view returns(uint256);
```
Comment (brendaloya) [inline contracts/interface/ITellor.sol:71]: Is this supposed to be bytes?

Comment (tkernell) [reply contracts/interface/ITellor.sol]: Why yes it is. Fixed!

**Comment by tkernell** [comment]
UsingTellor now includes an `isInDispute` getter. It retrieves the Governance address from Master, then queries `governance.getVoteRounds()`, and if the returned array's length is greater than 0, the value is under dispute.

**Pull request merged**